app/config.py

```diff
@@ -0,0 +1,19 @@
+GM_DROPDOWN_MENU_OPTIONS = [
+    {"label": "GCF ID", "value": "GCF_ID"},
+    {"label": "BGC Class", "value": "BGC_CLASS"},
+]
+
+GM_TEXT_INPUT_IDS_PLACEHOLDER = "1, 2, 3, ..."
+
+GM_DROPDOWN_BGC_CLASS_OPTIONS = [
+    {"label": "Alkaloid", "value": "ALKALOID"},
+    {"label": "NRP", "value": "NRP"},
+    {"label": "Polyketide", "value": "POLYKETIDE"},
+    {"label": "RiPP", "value": "RIPP"},
+    {"label": "Saccharide", "value": "SAACCHARIDE"},
+    {"label": "Terpene", "value": "TERPENE"},
+    {"label": "Other", "value": "OTHER"},
+    {"label": "Unknown", "value": "UNKNOWN"},
+]
+
+GM_DROPDOWN_BGC_CLASS_PLACEHOLDER = "Select one or more BGC classes"
```

Comment:
Duplication is not always bad if it helps readability. For example, I would prefer duplicating the `"1, 2, 3, ..."` and `"Select one or more BGC classes"`.